.github/workflows/clippy_changelog.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     - name: Check Changelog
       if: ${{ github.event_name == 'pull_request' }}
       run: |
-        body=$(curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -s "https://api.github.com/repos/rust-lang/rust-clippy/pulls/$PR_NUMBER" | \
+        body=$(curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" -s "https://api.github.com/repos/${{ github.repository_owner }}/rust-clippy/pulls/$PR_NUMBER" | \
```

Comment:
Should this be done with the repo name as well?

Reply:
Good point. I just changed this to use `github.repository` which contains the full `owner/repo` identifier anyway.